doc.go

```diff
@@ -0,0 +1,107 @@
+// Generated by running
+//  go generate github.com/gonum/matrix
+// DO NOT EDIT.
+
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package matrix provides common error handling mechanisms for matrix operations
+// in mat64 and cmat128.
+//
+// Overview
+//
+// This section provides a quick overview of the matrix package. The following
+// sections provide more in depth commentary.
+//
+// matrix provides:
+//  - Error type definitions
+//  - Error recovery mechanisms
+//  - Common constants used by mat64 and cmat128
+//
+// Errors
+//
+// The mat64 and cmat128 matrix packages share a common set of errors
+// provided by matrix via the matrix.Error type.
+//
+// Errors are either returned directly or used as the parameter of a panic
+// depending on the class of error encountered. As is usually the case in
+// Go code, returned errors indicate that a call was not able to complete
```

Comment:
Is it usually the case in Go code? Probably just delete the introductory clause, or change go to gonum.